File: tests/query/highlights/go/constructs.go
package main
//      ^ @namespace
type Point struct { X int }
//   ^ @type.definition
//                  ^ @variable.other.member
//                    ^ @type.builtin
func (p Point) Dist(count int) bool {
//             ^ @function.method.public
//                  ^ @variable.parameter
    return helper(count)
//         ^ @function
}

File: tests/query/highlights/python/constructs.py
import os
#      ^ @namespace
class Widget:
#     ^ @type
    def method(self, count: int) -> bool:
#       ^ @function
#              ^ @variable.builtin
#                    ^ @variable.parameter
#                           ^ @type.builtin
        return self.helper(count)
#                   ^ @function.method
